portingdb/templates/index.html

```diff
@@ -24,42 +64,68 @@ <h1>
         <div class="col-md-12">
             {{ progress_summary(status_summary, total_pkg_count) }}
         </div>
+        <div class="col-md-12">
+            <div class="main-summary">
+            <div class="summary-sentence">
+                <span class="big-percentage"
+                    style="color: {{ statuses['py3-only'].color }};"
+                >
+                    {{ (the_score * 100) | int }}%
+                </span>
+                <span class="stuffing">
+                    Python packages in {{ config['name'] }} support
+                </span>
+                <span class="callout">Python 3 only.</span>
+            </div>
+            <div class="summary-sentence">
+                <span class="big-percentage"
+                    style="color: {{ statuses['released'].color }};"
+                >{{ (py2_score * 100) | int }}%</span>
```

Comment:
...this includes dropped packages and hence is not true as in "X% Python packages in Fedora support Python 3."

Reply:
Good point. The goal changed now; "dropped" is not really "done". I'm keeping `DONE_STATUSES` as they were and adding a new set.